libelement/include/element/object.h

```diff
@@ -52,77 +54,77 @@ element_result element_delete_compilation_ctx(element_compilation_ctx** context)
  * @param[in] declaration       declaration to wrap
  * @param[out] output           result object
  *
- * @return ELEMENT_OK object compiled successfully
+ * @return ELEMENT_OK object created successfully
  * @return ELEMENT_ERROR_API_DECLARATION_IS_NULL declaration pointer is null
  * @return ELEMENT_ERROR_API_OUTPUT_IS_NULL output pointer is null
 */
 element_result element_declaration_to_object(const element_declaration* declaration, element_object** output);
 
 /**
- * @brief compiles selected object using the provided context
+ * @brief simplify selected object using the provided context
  *
  * if result was not OK but output object exists, call element_object_to_log_message
  * for more info or assign a log callback to the interpreter
  *
- * @param[in] object            compilation target
- * @param[in] context           compilation context
- * @param[out] output           result object
+ * @param[in] object            unsimplified object
+ * @param[in] context           object-model context
+ * @param[out] output           simplified object
  
- * @return ELEMENT_OK object compiled successfully
- * @return ELEMENT_ERROR_API_COMPILATION_CTX_IS_NULL context pointer is null
- * @return ELEMENT_ERROR_API_OBJECT_IS_NULL compilation target pointer is null
+ * @return ELEMENT_OK object simplified successfully
+ * @return ELEMENT_ERROR_API_OBJECT_MODEL_CTX_IS_NULL context pointer is null
+ * @return ELEMENT_ERROR_API_OBJECT_IS_NULL unsimplified object pointer is null
  * @return ELEMENT_ERROR_API_OUTPUT_IS_NULL output is null
  */
-element_result element_object_compile(
-    const element_object* object,
-    element_compilation_ctx* context,
-    element_object** output);
+element_result element_object_simplify(
+        const element_object* object,
```

Comment:
what's the reason for double indentation on some of these changes? it makes it inconsistent with the current indentation

Reply:
This wasn't a conscious choice - it's just how the IDE is configured by default and I must not have noticed. I can try and change my IDE settings to be consistent 👍 .